### PR TITLE
update the readme file 

### LINKED ISSUE #1: Update README file with some points
our repo should contain a README file with these three points -

WHY (describing the purpose)
WHAT (describing your site)
HOW (describing the journey to completion)

### PR DESCRIPTION
**Pull request by mo7amedehab97**
update the readme file relates #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+```shell
+git clone https://github.com/GSG-G11/news-team-2
+cd news-team-2
+go live server
+```
+
+## ✧Technology
+- HTML ➙ to build the structure of the pages 
+- CSS ➙ to style the app
+
```
Comment (Jehad91) [inline README.md:44]: add JS to toggle bar button on mobile screen

**Review by Jehad91 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mo7amedehab97) [reply README.md:44]: ok, I will add it :)

Comment (AhmedQeshta) [reply README.md:44]: I suggest. To add that we used Flexbox, use animations, and ES6

**Review by karamzomlot — APPROVED** [review]

**Review by karamzomlot — APPROVED** [review]

**Review by AhmedQeshta — APPROVED** [review]

**Review by Jehad91 — COMMENTED** [review]
good

**Pull request merged**